test/integration/singlecluster/tas/tas_test.go

```diff
@@ -520,6 +520,120 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 				})
 			})
 
+			ginkgo.It("should respect TAS usage by admitted workloads after reboot; second workload created after reboot", func() {
```

Comment:
I think it is safe to drop this test. The sibling test was much more sensitive in my testing.

Reply:
Dropped